dimod/core/polysampler.py

```diff
@@ -51,55 +57,44 @@ def sample_poly(self, polynomial, **kwargs):
         """Sample from a higher-order polynomial."""
         pass
 
-    def sample_hising(self, h, J, **kwargs):
+    def sample_hising(self, h: typing.Dict[Variable, Bias],
```

Comment:
also, strictly speaking this should be `typing.Mapping` rather than `typing.Dict`. E.g. `bqm.linear` is a mapping but not a dict.

Reply:
Though if you return a dict, you _should_ use `typing.Dict`, since it's more specific. Isn't typing fun?